OKTA-750492 SCIM deprecation (#5010)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/oin-lifecycle-mgmt-overview/index.md

    content: Learn how to build a lifecycle management app integration for the Okta Integration Network.
---

> Lifecycle management refers to the process of provisioning and deprovisioning app access as a user moves through an org. Onboarding, role change, transfer, and offboarding employee processes require provisioning user accounts in workforce apps. It also requires enforcing corporate security policies in a timely manner.

Reviewer: I believe we capitalize Workforce as it's an Okta thing.

```suggestion
Lifecycle management refers to the process of provisioning and deprovisioning app access as a user moves through an org. Onboarding, role change, transfer, and offboarding employee processes require provisioning user accounts in Okta Workforce apps. It also requires enforcing corporate security policies in a timely manner.
```

Reviewer: Hmm - the fact that we read it two separate ways means it needs to be clarified. 👍 

(And the fact that Workday as an HR app is involved is also bringing additional confusion sauce to the table.)

I'd suggest changing it to just have "require provisioning user accounts in Okta and other third-party apps"?

</div>

Okta provides two methods to integrate lifecycle management to your app. You can use the [System for Cross-domain Identity Management](https://developer.okta.com/docs/api/openapi/okta-scim/guides/) (SCIM) protocol to manage user accounts from Okta. You can also use the [Okta Worfklows Connector Builder](https://help.okta.com/okta_help.htm?type=wf&id=ext-connector-builder) to create automated identity processes in [Okta Workflows](https://help.okta.com/okta_help.htm?type=wf).

After completing your Okta lifecycle management integration, publish it in the [Okta Integration Network (OIN)](https://www.okta.com/integrations/) for your customers to benefit from secure provisioning automation and meet compliance requirements. The OIN is a collection of pre-built app integrations covering numerous use cases. Join the community that customers trust to exchange secure authentication between users, devices, and apps.

## Why build a lifecycle management integration with Okta?

| &nbsp; | &nbsp; |
| ------ | ------ |

> | **Automate onboarding/offboarding and win larger customers** | After an org reaches a certain size, manually provisioning accounts and getting users set up are time-consuming and a bottleneck for your product's adoption. Okta lifecycle management integration automates mundane tasks for your customers, which reduces the total cost of ownership and removes barriers to end user activation. |

Reviewer: ```suggestion
| **Automate onboarding/offboarding and win larger customers** | After an org reaches a certain size, manually provisioning accounts and getting users set up is a time-consuming process and a bottleneck for your product's adoption. Okta lifecycle management integration automates mundane tasks for your customers, which reduces the total cost of ownership and removes barriers to end user activation. |
```

> | **Automate onboarding/offboarding and win larger customers** | After an org reaches a certain size, manually provisioning accounts and getting users set up is a time-consuming process and a bottleneck for your product's adoption. Okta lifecycle management integration automates mundane tasks for your customers, which reduces the total cost of ownership and removes barriers to end user activation. |

Reviewer 2: why did you remove the hyphen here?
```suggestion
| **Automate onboarding/offboarding and win larger customers** | After an org reaches a certain size, manually provisioning accounts and getting users set up is a time-consuming process and a bottleneck for your product's adoption. Okta lifecycle management integration automates mundane tasks for your customers, which reduces the total cost of ownership and removes barriers to end-user activation. |
```

Author: Because Acrolinx got mad when it was in there: 
<img width="303" alt="Screenshot 2024-08-12 at 3 28 37 PM" src="https://github.com/user-attachments/assets/ad8890e1-e8a2-4373-a4d4-cd6b5988be73">

> | **Automate onboarding/offboarding and win larger customers** | After an org reaches a certain size, manually provisioning accounts and getting users set up is a time-consuming process and a bottleneck for your product's adoption. Okta lifecycle management integration automates mundane tasks for your customers, which reduces the total cost of ownership and removes barriers to end-user activation. |

Reviewer 2: ```suggestion
| **Automate onboarding/offboarding and win larger customers** | After an org reaches a certain size, manually provisioning accounts and getting users set up is a time-consuming process and a bottleneck for your product's adoption. Okta lifecycle management integration automates mundane tasks for your customers, which reduces the total cost of ownership and removes barriers to end user activation. |
```

| **Secure access to your app** | Secure access to your app. Integrating with Okta ensures that end users can access your app when they join an org. Access is automatically removed as they leave the org to prevent embarrassing data loss and dangling access. |
| **Meet compliance requirements** | Help your customers generate logs of access changes to comply with security policies and audits. |
| **Increase adoption of your product and sell more licenses** | Okta lifecycle management integration helps your customers use SaaS licenses and ultimately purchase more as the org grows or use expands. |

## Choose how to implement your integration

| &nbsp; |  System for Cross-domain Identity Management (SCIM) |  Connector Builder |
| ------ | :------------------- | :----------------------- |

> | **Description** | [SCIM](https://scim.cloud) is the industry standard protocol for managing users and groups in cloud-based systems. It handles basic operations like create, read, update, delete, and search. | [Connector Builder](https://help.okta.com/okta_help.htm?type=wf&id=ext-connector-builder) is a no-code, if-this-then-that logic builder. Okta customers can use it to automate custom or complex employee onboarding and offboarding flows in your app. Publishing a Connector with Okta allows your customers to deeply integrate your product as part of their lifecycle management flows. |

Reviewer: ```suggestion
| **Description** | [SCIM](https://scim.cloud) is the industry standard protocol for managing users and groups in cloud-based systems. It handles basic operations like create, read, update, delete, and search. | [Connector Builder](https://help.okta.com/okta_help.htm?type=wf&id=ext-connector-builder) is a no-code development interface that creates custom connections to API endpoints along with data manipulation flows. You can then package these flows with authentication and branding as connectors in the OIN. Publishing a complete connector allows your customers to deeply integrate your product as part of their lifecycle management process in Okta Workflows. |
```

| **Technology** | <ul><li>JSON-based REST API server implementation</li> <li>Okta supports outbound SCIM requests to your SCIM API service</li> <li>See [SCIM Protocol](https://developer.okta.com/docs/api/openapi/okta-scim/guides/)</li></ul> | <ul><li>Low code development environment</li> <li>See [Okta Workflows](https://help.okta.com/okta_help.htm?type=wf)</li></ul>|

> | **Benefits** | <ul><li>Covers the most common lifecycle management needs related to onboarding/offboarding</li> <li>SCIM standard used throughout the industry</li> </ul> | <ul><li>Customers can create highly custom automation with your product without writing code</li> <li>Your product is tightly integrated with Okta</li> </ul>|

Reviewer: ```suggestion
| **Benefits** | <ul><li>Covers the most common lifecycle management needs related to onboarding/offboarding</li> <li>SCIM is an industry standard protocol</li> </ul> | <ul><li>Customers can create highly custom automated flows using your product without writing code</li> <li>Your product is tightly integrated with Okta</li> </ul>|
```

<Card href="/docs/guides/scim-provisioning-integration-overview/" headerImage="/img/icons/scim.svg" cardTitle="System for Cross-domain Identity Management" :showFooter=false>Build a SCIM provisioning integration</Card>

> <Card href="https://help.okta.com/okta_help.htm?type=wf&id=ext-connector-builder" headerImage="/img/icons/WEB_Icon_Platform_Workflows_40x40.svg" cardTitle="Connector Builder" :showFooter=false>Build a Workflows integration</Card>

Reviewer: ```suggestion
<Card href="https://help.okta.com/okta_help.htm?type=wf&id=ext-connector-builder" headerImage="/img/icons/WEB_Icon_Platform_Workflows_40x40.svg" cardTitle="Connector Builder" :showFooter=false>Build a Workflows connector</Card>
```
